sympy/core/add.py

```diff
@@ -387,19 +387,17 @@ def _eval_power(self, e):
         elif e.is_Number and abs(e) != 1:
             # handle the Float case: (2.0 + 4*x)**e -> 2.**e*(1 + 2.0*x)**e
```

Comment:
This should be `(2.0 + 4*x)**e -> 4**e*(0.5 + x)**e`

Reply:
Done.

Reply:
I made one more comment about the issue test case. Other than that it looks like the issue is resolved. Thank for your work on this.